src/test/test-piece-kicks.gd

```diff
@@ -58,11 +58,17 @@ func _assert_kick() -> void:
 
 
 func _kick_piece() -> Vector2:
-	var result := Vector2.ZERO
+	var result
 	_from_piece = _create_active_piece(from_grid)
 	_to_piece = _create_active_piece(to_grid)
-	if not _from_piece.can_move_piece_to(funcref(self, "_is_cell_blocked"), _from_piece.pos, _to_piece.orientation):
+	if _from_piece.can_move_piece_to(funcref(self, "_is_cell_blocked"), _from_piece.pos, _to_piece.orientation):
+		# no kick necessary
+		result = Vector2.ZERO
+	else:
 		result = _from_piece.kick_piece(funcref(self, "_is_cell_blocked"), _from_piece.pos, _to_piece.orientation)
+		if not result:
+			# the tests need to distinguish between a piece rotating in place, and failing to rotate
+			result = Vector2(-99, -99)
```

Comment:
Worth mentioning in commit comment

Reply:
Extract a constant

Reply:
Mentioned in commit comment: `Changed test-piece-kicks framework to distinguish between a piece which didn't rotate, and a piece which rotated but didn't move.`

Extracted a constant, `FAILED_KICK`